.develop/devspace.yaml

```diff
@@ -78,6 +78,14 @@ dev:
 profiles:
   - name: dev
     patches:
+      # replace the image pull policy for dev (but only for init/manager!) to IfNotPresent so that
```

Comment:
@carlmontanari why do we do this only for the manager, but not for launcher?

Reply:
we can set launcher too, but I think the "normal case" will be devspace for dev'ing on manager and the launcher will just be pulled normally. we can absolutely add it though! (but it wont prevent devspace/manager from starting so that part is at least solved for now)